tools/cjxl_main.cc

```diff
@@ -144,7 +144,10 @@ struct CompressArgs {
     cmdline->AddOptionValue(
         'e', "effort", "EFFORT",
         "Encoder effort setting. Range: 1 .. 9.\n"
-        "    Default: 7. Higher number is more effort (slower).",
+        "    Default: 7. Higher numbers allow more computation "
+        "at the expense of time.\n"
+        "    For lossless, generally it will produce smaller files at the same quality.\n"
```

Comment:
This sentence is weird. Just drop the "at the same quality"?

Reply:
Yeah, I did think it was a bit redundant, removing it now

Reply:
@jonsneyers @jonnyawsom3 The original suggestion https://github.com/libjxl/libjxl/pull/2552#discussion_r1231633026 described the behavior irrelevant of the mode. Indeed, qualifying it here as a feature of lossless makes the phrase redundant, while the original sentence was more informative.

Similarly, the suggestion that allowing "more computation" comes "at the expense of time" is redundant (when true). The original suggestion rather highlighted the computation/size tradeoff.